### PR TITLE
Fix download button in the admin when the latest version is selected

### PR DESCRIPTION
**Pull request by faebebin**
To be tested ...

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-2744 Quickfix Admin Projects download latest version button](https://app.clickup.com/t/2192114/QF-2744)

File: docker-app/qfieldcloud/core/templates/admin/project_files_widget.html [hunk 1]
```diff
@@ -243,10 +243,12 @@
...
-                            const buildApiUrlWithPath = () => buildApiUrl(pathToFile, {
-                                version: $versionsSelect.value,
-                            });
-                            window.open($versionsSelect.value ? buildApiUrlWithPath() : pathToFile);
+                            const version_id = $versionsSelect.value || file.versions[0]?.version_id;
+                            if (version_id) {
+                                window.open(buildApiUrl(pathToFile, {version: version_id}))
+                            } else {
+                                window.open(pathToFile)
+                            }
```
Comment (suricactus) [inline docker-app/qfieldcloud/core/templates/admin/project_files_widget.html:251]:
```suggestion
                            const versionId = $versionsSelect.value || file.versions[0]?.version_id;
                            if (versionId) {
                                window.open(buildApiUrl(pathToFile, {version: versionId}))
                            } else {
                                window.open(pathToFile)
                            }
```

Comment (suricactus) [reply docker-app/qfieldcloud/core/templates/admin/project_files_widget.html:251]: Allowed myself to add a small rewording comment on variable naming.

**Review by suricactus — APPROVED** [review]
I like the simple and pragmatic way of dealing with the task within the PR. I would prefer we stick to pascalCase within the code.

**Pull request closed without merging**